modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/rest-openapi.yaml

```diff
@@ -1120,6 +1120,30 @@ paths:
                     description:
                         ""
             tags: ["Instance"]
+    "/processes/{processId}/last-checked-date":
```

Comment:
Hey @MarinhoFeliphe, this is not a generated change. Please split this commit and put the generate commit as one of the last ones. Please.